qt/aqt/main.py

```diff
@@ -1075,7 +1075,7 @@ def maybe_auto_sync_media(self) -> None:
         if self.can_auto_sync():
```

Comment:
Looks like an inverted check bug?

Reply:
You're right, but it's doubly-wrong - it's looking at the auto-sync setting instead of the periodic sync setting. I'll refactor.